### PR TITLE
Styling for home, login and signup page, etc.,

### PR DESCRIPTION
**Pull request by varun-em**
- Icons for home page
- Icons for dashboard cards
- Dark background with image for login and signup screens
- Created reusable components using styled components

image
image
image
image
image

### CODE REVIEW

File: src/components/Dashboard/Dashboard.js [hunk 1]
```diff
@@ -59,124 +65,120 @@ export const Dashboard = () => {
...
+                        <div className="row g-0">
+                          <div className="col-md-4 bg-white bg-opacity-10 d-flex justify-content-center align-items-center">Icon</div>
+                          <div className="col-md-8">
+                            <div className="card-body">
+                              <h6 className="card-title">Agent Verification Pending</h6>
+                              <h6 className="mb-0">21</h6>
                             </div>
                           </div>
                         </div>
-                        <div className="col-3 mb-3">
```
Comment (prashant-thakar) [inline src/components/Dashboard/Dashboard.js:150]: Remove the commented code

File: src/components/Dashboard/Dashboard.js [hunk 2]
```diff
@@ -59,124 +65,120 @@ export const Dashboard = () => {
...
+                      number={drivers}
+                      role={"button"} onClick={(e) => setActiveScreen("drivers")}
+                    />
+                    <DashboardGridItem
+                      icon={<VehicleIcon />}
+                      title="Total Vehicles"
+                      number={vehicles}
+                      role={"button"} onClick={(e) => setActiveScreen("drivers")}
+                    />
+                    {/* <DashboardGridItem
```
Comment (prashant-thakar) [inline src/components/Dashboard/Dashboard.js:128]: Remove the commented code

File: src/components/LoginModule/SignInPassword/SignInPassword.js [hunk 3]
```diff
@@ -3,6 +3,10 @@ import { useEffect, useState } from "react";
 import { Col, Container, Row } from "react-bootstrap";
 import { isAuthenticated } from "../../../core/common.functions";
 import { AppRoutes } from "../../../core/constant";
+import { PrimaryButton } from "../../../shared/buttons/PrimaryButton";
```
Comment (prashant-thakar) [inline src/components/LoginModule/SignInPassword/SignInPassword.js:6]: Put this all in to index.js and do export once { list of import } index.js

File: src/shared/links/PrimaryButtonLink.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
+import { Link } from "react-router-dom";
```
Comment (prashant-thakar) [inline src/shared/links/PrimaryButtonLink.js:1]: User Bootstrap btn-primary class and modify bootstrap btn scss snipet in _custome.bootstrap.scss on app.js

File: src/shared/links/PrimaryButtonLink.js [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+import styled from "styled-components";
+
+export const PrimaryButtonLink = styled(Link)`
```
Comment (prashant-thakar) [inline src/shared/links/PrimaryButtonLink.js:4]: we won't need this link class low level component just remove and user btn-link class of bootstrap

File: src/components/Dashboard/Dashboard.js (hunk 2 above)
Comment (varun-em) [reply src/components/Dashboard/Dashboard.js]: Removed

File: src/components/LoginModule/SignInPassword/SignInPassword.js (hunk 3 above)
Comment (varun-em) [reply src/components/LoginModule/SignInPassword/SignInPassword.js]: Done for icons, deleted buttons folder and updated custom.scss

File: src/shared/links/PrimaryButtonLink.js (hunk 4 above)
Comment (varun-em) [reply src/shared/links/PrimaryButtonLink.js]: Done as requested

File: src/shared/links/PrimaryButtonLink.js (hunk 5 above)
Comment (varun-em) [reply src/shared/links/PrimaryButtonLink.js]: Removed it as asked

File: src/App.scss [hunk 6]
```diff
@@ -1,5 +1,5 @@
-@import "./stylesheets/custom.scss";
 @import "~bootstrap/scss/bootstrap";
+@import "./stylesheets/custom.scss";
```
Comment (prashant-thakar) [inline src/App.scss:2]: Move this upside; this is contained overwrite bootstrap variable

Comment (varun-em) [reply src/App.scss:2]: No, it doesn't work if we change the order, please check by running it on your side

Comment (varun-em) [reply src/App.scss]: Created new file instead, hope that works

**Pull request merged**